packages/redux/package.json

```diff
@@ -27,6 +27,7 @@
     "start": "cross-env NODE_ENV=development npm run watch"
   },
   "dependencies": {
+    "@deephaven/jsapi-utils": "file:../jsapi-utils",
```

Comment:
The other `@deephaven` package dependencies list a version and not the local file

Reply:
I think we use `file` everywhere now.

Reply:
Just to clarify - we use `file` in all `dependencies` and `devDependencies`, but not for `peerDependencies`.
When the package is published, the `package.json` generated converts the `file:....` to the actual version number. You can confirm this by doing an `npm pack @deephaven/redux@latest` and checking the `package.json`. It has the actual version number of the deps instead of keeping the file tag.
I don't quite understand why it converts deps and devDeps but does not convert peerDeps.

Reply:
Another note - I converted to `file:...` because on version bump, lerna would bump these versions and then `npm install` wouldn't work - because the version hadn't been published yet!